hello.go

```diff
@@ -173,6 +173,13 @@ func main() {
 		tmpl.Execute(w, data)
 	})
 
+	http.HandleFunc("/info", func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Content-Type", "application/json")
+		encoder := json.NewEncoder(w)
+		encoder.SetIndent("", "  ")
```

Comment:
Why do you want to use `SetIndent` on `encoder`? Just for cosmetic?
It's not better to use `jq` locally and keep `json` "compressed"?

Reply:
Works for me. Removed the `SetIndent`.